docs/en/ingest-management/elastic-agent/elastic-agent-container.asciidoc

```diff
@@ -59,6 +68,8 @@ docker run \
   --rm docker.elastic.co/beats/elastic-agent:{version}
 ----
 
+NOTE: Replace `docker.elastic.co/beats/elastic-agent` with `docker.elastic.co/beats/elastic-agent-complete` and use the `elastic-agent` user instead of root to run Synthetics Browser tests. Synthetics tests cannot run under the root user. See {observability-guide}/synthetics-quickstart-fleet[Synthetics Fleet Quickstart] for more information.
```

Comment:
```suggestion
NOTE: Replace `docker.elastic.co/beats/elastic-agent` with `docker.elastic.co/beats/elastic-agent-complete` and use the `elastic-agent` user instead of root to run Synthetics Browser tests. Synthetics tests cannot run under the root user. See {observability-guide}/synthetics-quickstart-fleet.html[Synthetics Fleet Quickstart] for more information.
```